geps/gep-1742.md

```diff
@@ -184,11 +187,14 @@ sequenceDiagram
         Note right of U: (between read operations)<br/>proxy_read_timeout<br/>grpc_read_timeout
     U->>P: Finishes Response
     deactivate U
+    activate P
     P->>C: Starts Response
     P->>C: Finishes Headers
     P->>C: Finishes Response
-    Note right of P: Repeat if connection sharing
-    Note Right of U: keepalive_timeout (if keepalive enabled)
+    deactivate P
+    Note left of P: send_timeout (only between two successive write operations)
+    Note right of P: Repeat if connection is shared until server's keepalive_timeout is hit
```

Comment:
I think this note based on its position still refers to the connection between Nginx and the Upstream. So I suggest updating it description, removing  keepalive_timeout since it's already mentioned on the right of U:
```suggestion
    Note right of P: Repeat if connection is shared
```


Would it make sense to add `server's keepalive_timeout` to the left of C after "Finishes Response"? This will be similar to Envoy's and HAProxy's diagrams.

Reply:
@pleshakov 
Thanks for your suggestion! Would you review my latest change?